### PR TITLE
[2.3] [CS] MethodSeparationFixer - findings.

### PR DESCRIPTION
**Pull request by SpacePossum**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| License | MIT |

Th goal of this PR is _not_ to have it merged, but to see if the `MethodSeparationFixer` (https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1426) is suitable for SF level/configuration in the PHP-CS-Fixer (and thereby might later become part of `fabbot.io`).

### CODE REVIEW

File: src/Symfony/Component/HttpFoundation/Response.php [hunk 1]
```diff
@@ -977,6 +977,7 @@ public function isNotModified(Request $request)
     }
 
     // http://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html
+
```
Comment (fabpot) [inline src/Symfony/Component/HttpFoundation/Response.php]: We should probably move the URL in the docblock instead.

**Comment by fabpot** [comment]
:+1:

**Comment by keradus** [comment]
:+1:

**Comment by fabpot** [comment]
Thank you @SpacePossum.

**Pull request merged**